pkg/cli/root.go

```diff
@@ -46,6 +46,7 @@ https://github.com/replicate/cog`,
 		newPushCommand(),
 		newRunCommand(),
 		newTrainCommand(),
+		newServeCommand(),
```

Comment:
```suggestion
		newServeCommand(),
		newTrainCommand(),
```

We must serve the Great Alphabet 🙇